Assets/Scripts/UI/ColorPalette/ColorPalette.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace PotentialRobot.UI.ColorPalettes
+{
+    [CreateAssetMenu(menuName = "ColorPalette/Create new Color Palette", fileName = "ColorPalette_name")]
+    public class ColorPalette : ScriptableObject, IColorPalette
+    {
+        [field: SerializeField]
+        public string PaletteType { get; private set; }
```

Comment:
What's this?

Reply:
Not required anymore, so I'll remove it entirely.